packages/datatrak-web/src/routes/Routes.tsx

```diff
@@ -37,6 +39,16 @@ const LoggedInRedirect = ({ children }) => {
   return <Navigate to={user.projectId ? ROUTES.HOME : ROUTES.PROJECT_SELECT} replace={true} />;
 };
 
+const ReportsRedirect = () => {
+  const { isLoading, data: user } = useUser();
```

Comment:
```suggestion
  const { isLoading, data: user } = useCurrentUser();
```
If we used useCurrentUser then we wouldn't have to worry about loading state but even better would be to either combine this check into the Private Route as some kind of Admin check or extend the private route for Admin checking and then remove the ReportsRedirect component entirely.